tests/foreman/ui/test_contentviews.py

```diff
@@ -977,6 +976,27 @@ def test_cv_clone_within_same_env(self):
         @status: Manual
 
         """
+        repo_name = gen_string('alpha', 8)
+        cv_name = gen_string('alpha', 8)
+        copy_cv_name = gen_string('alpha', 8)
+        with Session(self.browser) as session:
+            self.setup_to_create_cv(repo_name=repo_name)
+            # Create content-view
+            make_contentview(session, org=self.org_name, name=cv_name)
+            self.assertIsNotNone(self.content_views.search(cv_name))
+            # Add repository to selected CV
+            self.content_views.add_remove_repos(cv_name, [repo_name])
+            self.assertIsNotNone(self.content_views.wait_until_element
```

Comment:
This should look better

```python
self.assertIsNotNone(
    self.content_views.wait_until_element(
        common_locators["alert.success"])
)
```

Having the code as left aligned as possible increase legibility